hw/usb/tinyusb/cdc_console/syscfg.yml

```diff
@@ -25,5 +25,12 @@ syscfg.defs:
 syscfg.vals:
     USBD_CDC_CONSOLE: 1
 
+syscfg.defs:
+    CONSOLE_USB_CDC_SYSINIT_STAGE:
+        description: >
+          Tiny USB needs to be initialized before Console USB CDC is \
+          initialized
```

Comment:
this lacks description of what it configures

Reply:
updated